templates/bundles/DemosPlanCoreBundle/UI/form/label.html.twig

```diff
@@ -53,9 +53,8 @@
     {% endif %}
 </label>
 {% if tooltip|default %}
-    <i
-        class="{{ 'fa fa-question-circle absolute mt-0.5 top-0 right-0'|prefixClass(omitPrefix) }}"
-        :aria-label="Translator.trans('contextual.help')"
-        v-tooltip="'{{ tooltip|trans }}'">
-    </i>
+    {% include '@DemosPlanCore/Extension/contextual_help.html.twig' with {
+        helpText: tooltip|trans,
+        cssClasses:'mt-0.5 top-0 right-0'
```

Comment:
```suggestion
        cssClasses:'absolute mt-0.5 top-0 right-0'
```

Reply:
![Screenshot 2024-04-22 at 10-12-50 Stellungnahme Details Test_2 9 2_I Bauleitplanung Online](https://github.com/demos-europe/demosplan-core/assets/59571234/ee9f6428-6729-4b82-a186-171105ba9b0c)
When use absolute!

Reply:
then You have to add `relative` to the parent element

Reply:
I think it would be better to move the icon within the label element.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/2970/commits/e3d1d2af0b36e3750c5a3debb63dc21aaf8e741d